io.gitlab.Turtlico.json

```
@@ -29,79 +33,85 @@
+                    "sources": [
+                        {
+                            "type": "archive",
+                            "url": "https://prdownloads.sourceforge.net/tcl/tk8.6.12-src.tar.gz",
+                            "sha256": "12395c1f3fcb6bed2938689f797ea3cdf41ed5cb6c4766eec8ac949560310630",
+                            "x-checker-data": {
+                                "type": "anitya",
+                                "project-id": 4941,
+                                "stable-only": true,
+                                "url-template": "https://prdownloads.sourceforge.net/tcl/tk$version-src.tar.gz"
+                            }
+                        }
+                    ]
                 }
-            ],
-            "subdir" : "unix",
-            "config-opts": [
-                "--with-x",
-                "--enable-threads"
-            ],
-            "post-install" : [
-                "chmod 755 /app/lib/libtk8.6.so"
             ]
         },
         {
-            "name" : "python3",
-            "config-opts" : [
-                "--enable-ipv6",
-                "--with-threads",
-                "--with-lto",
-                "--with-system-ffi",
-                "--with-system-libmpdec",
-                "--with-system-expat"
-            ],
-            "cleanup" : [
-                "/bin/2to3*",
-                "/bin/easy_install*",
-                "/bin/idle*",
-                "/bin/python*-config",
-                "/bin/pyvenv*",
-                "/bin/pydoc*",
-                "/include",
-                "/share",
-                "/lib/pkgconfig",
-                "/lib/python*/config-*",
-                "/lib/python*/test",
-                "/lib/python*/distutils/tests",
-                "/lib/python*/ctypes/test",
-                "/lib/python*/sqlite3/test",
-                "/lib/python*/lib2to3/tests",
-                "/lib/python*/tkinter/test",
-                "/lib/python*/unittest/test",
-                "/lib/python*/lib-dynload/_*_test.*.so",
-                "/lib/python*/lib-dynload/_test*.*.so",
-                "*.pyc",
-                "__pycache__",
-                "*.exe"
+            "name": "python3-turtle",
+            "buildsystem": "simple",
+            "build-commands": [
+                "cp turtle.py ${FLATPAK_DEST}/lib/python3.9/site-packages"
             ],
-            "sources" : [
+            "sources": [
                 {
-                    "type" : "archive",
-                    "url" : "https://www.python.org/ftp/python/3.8.6/Python-3.8.6.tar.xz",
-                    "sha256" : "a9e0b79d27aa056eb9cce8d63a427b5f9bab1465dee3f942dcfdb25a82f4ab8a"
+                    "type": "file",
+                    "url": "https://raw.githubusercontent.com/python/cpython/3.9/Lib/turtle.py",
+                    "dest-filename": "turtle.py",
+                    "sha256": "fd87a4dad42efda9ed3553d226f229b5c3725f5e6c8805831864640d9b628fd1"
```

```suggestion
                    "url": "https://raw.githubusercontent.com/python/cpython/v3.9.9/Lib/turtle.py",
                    "dest-filename": "turtle.py",
                    "sha256": "fb3209395b13b2a45a3371606d604079e66da241be4453b6034a85960c186182"
```

It's better to use it from a tag, because from a branch, build will break at one point.